tests/chainer_tests/functions_tests/connection_tests/test_n_step_lstm.py

```diff
@@ -103,28 +103,15 @@ def check_forward(
         testing.assert_allclose(cy.data, e_cy, rtol=1e-4, atol=1e-4)
 
     def test_forward_cpu(self):
-        self.check_forward(self.hx, self.cx, self.xs, self.ws, self.bs, False)
-
-    def test_forward_cpu_volatile(self):
-        self.check_forward(self.hx, self.cx, self.xs, self.ws, self.bs, True)
+        self.check_forward(self.hx, self.cx, self.xs, self.ws, self.bs)
```

Comment:
To keep the strength of tests, how about testing the propagation with both `no_backprop_mode` and `enable_backprop_mode`?